Library/Homebrew/api/formula.rb

```diff
@@ -10,9 +10,9 @@ module API
     #
     # @api private
     module Formula
-      class << self
-        include Cachable
+      extend Cachable
 
+      class << self
```

Comment:
Might as well use `private_class_method` and move everything to `def self.*` here.

Reply:
Can be done in a follow-up PR though.

Reply:
I didn't think that'd work but it seems fine. Good idea!

Reply:
Sorry, I got confused. Yeah, I'll update that too.

Reply:
Might not be worth killing the git blame though. I'll do it anyway and see if anyone cares.

Reply:
Addressed in https://github.com/Homebrew/brew/pull/16746/commits/bea2dc65fec2ef41c90918a541fef6deb3e15142